pkg/skaffold/build/local.go

```diff
@@ -64,6 +64,12 @@ func NewLocalBuilder(cfg *v1alpha2.BuildConfig, kubeContext string) (*LocalBuild
 	return l, nil
 }
 
+func (l *LocalBuilder) Labels() map[string]string {
+	return map[string]string{
+		constants.Labels.Builder: "local",
```

Comment:
nit:
add some context from the docker api

`"docker-api-version": api.ServerVersion(ctx)`